rename_css.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+
+set -e
+
+function sed_in_place {
+    quote="'"
+    expr=$1
+    echo "Expression is: $quote$expr$quote -----"
+    shift 1
+    files=$*
+    for file in $files
+    do
+        backup=$file.bak
+        sed "$expr" $file > $file.bak
+        if [ $? -eq 0 ]; then
+            mv $file.bak $file
+        else
+            echo "didn't work on $file"
+            break
+        fi
+    done
+}
+
+css_files=$(find site -name '*.css')
+echo "Here are all the CSS files:" $css_files
+
+all_html_files=$(find site -name '*.html')
```

Comment:
Theoretically, CSS can include other CSS, but I don't think that is the case with this site.